### ISSUE TITLE
Hard time replicating MAML results from the example repos

### ISSUE THREAD

**Issue by deltaskelta** [tristandeleu/pytorch-meta#20]
I am trying to replicate the paper results for MAML after looking at many implementations. I have been struggling to get to 98-99 percent accuracy while using `--first-order` derivatives as they say is possible in the paper. 

I thought it must be something with my implementation so I downloaded and ran the example in this repo and I noticed the same thing. 

5-way 1-shot on omniglot converges to 99% train accuracy very quickly but when I add the `--first-order` option it converges very slowly and hangs around 90% ish accuracy. 

Does the `--first-order` option need a much longer training time, or is it possible that there could be a bug in the example or somewhere in the library which is making this happen?

**Comment by tristandeleu** [comment]
The implementation in the example folder is only meant to be an example of how to use Torchmeta, but this is not necessarily a robust implementation of MAML. I would suggest to look at https://github.com/tristandeleu/pytorch-maml/ which contains a more complete version of MAML (which still uses Torchmeta).

However I don't think I did enough experiments with `first_order=True`, so if you see the same behavior let me know.

**Comment by deltaskelta** [comment]
Ok I will let you know if I find something. Thanks

**Comment by renesax14** [comment]
> Ok I will let you know if I find something. Thanks

Did you end up replicating MAML?

If yes, how long did it take you to run experiments?

It seems 60K episodes takes 16hours for me...is that true for you?
